_pages/tickets.md

```diff
@@ -7,46 +7,56 @@ published: true
 title: Tickets
 ---
 
-Ticket sales will be open soon. 
-
-Keep an eye on our [News](/news) page and follow us on [Twitter](https://twitter.com/DjangoConEurope) for updates.
-
-{% comment %}
+**Ticket sales are now open!  All tickets include access to all 3 conference days (29th - 31st May) and the 2 sprints days (1st - 2nd June).**
 
 # What ticket should I buy?
 
 If you are paying for your own ticket, you should buy an **Individual** ticket. If your company 
-is paying, you should buy a **Corporate** ticket. For those who can't afford the Individual ticket,
-a **discounted** ticket is available; see below. 
+is paying, you should buy a **Corporate** ticket. 
 
 You can also buy a **supporter** ticket if you want to go above and beyond and support our mission!
 These tickets will allow us to offer discounted and free tickets to people who couldn't otherwise
 afford to attend.
 
+If you are a student or without income, a **discounted** ticket is available. This ticket type does not cover our costs for the conference, but we’re able to offer it thanks to the generosity of our
+sponsors and attendees who purchase supporter tickets.
+
 <a class="button" href="{{ site.ticket_link }}" target="_blank">Buy Tickets Now!</a>
 
+## What's included?
+
+All tickets include access to all 3 conference days (29th - 31st May) and the 2 sprints days (1st - 2nd June).
+
+The following refreshments are included on the 3 conference days: mid-morning coffee and snack, lunch, afternoon coffee and snack.  
+
+Tickets for the social event on 29th May can be purchased as an add-on to any conference ticket at a small additional cost.
+
+Childcare for the 3 conference days is free, but please let us know about your childcare requirements
```

Comment:
@rebkwok maybe we could add a link to the about page with the childcare reference?

Reply:
Done; don't merge just yet though!  Pretix isn't quite ready.